src/main/java/com/cinema/Main.java

```diff
@@ -68,25 +70,25 @@ public static void main(String[] args) throws AuthenticationException {
         sessionFastAndFurious2.setShowTime(LocalDateTime.now().minusDays(2));
         sessionService.add(sessionFastAndFurious2);
 
-        System.out.println("------------------Find Fast And Furious Sessions--------------");
+        logger.info("------------------Find Fast And Furious Sessions--------------");
         sessionService.findAvailableSessions(fastAndFurious.getId(),
                 LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(logger::info);
         System.out.println("------------------Find Sully Sessions--------------------------");
         sessionService.findAvailableSessions(sully.getId(), LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(logger::info);
 
-        System.out.println("------------------Users Check--------------------------");
+        logger.info("------------------Users Check--------------------------");
         AuthenticationService authService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         authService.register("new1@gmail.com", "1234");
         authService.register("new2@gmail.com", "1234");
         User new1 = authService.login("new1@gmail.com", "1234");
         User new2 = authService.login("new2@gmail.com", "1234");
```

Comment:
let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block

Reply:
done